language/stdlib/modules/libra_configs.move

```diff
@@ -0,0 +1,40 @@
+address 0x0:
+module LibraConfig {
+    use 0x0::Transaction;
+    use 0x0::LibraSystem;
+
+    // A generic singleton resource that holds a value of a specific type.
+    resource struct T<U: copyable> { payload: U }
```

Comment:
Perhaps `U` should be called `Value` or `Config`? The convention we have used elsewhere is to have descriptive names for generic type parameters.